### ISSUE TITLE
Which is the latest stable version?

### ISSUE THREAD

**Issue by jnnkB** [pinry/pinry#217]
The latest release you have is 2.0.1 from Jan 22, 2019. Is this the version that should be run? Because if you follow the guide https://docs.getpinry.com/install-with-docker/, it installs pinry from the master branch – which didn't seem stable when I tried.

**Comment by winkidney** [comment]
Thanks for your feedback.

TL;DR: Pinry has no `stable version` and we only have one release named `master`.

Long answer:

+ 1.x is just a history version and no further maintaining will be done on it.
+ we have no strict version management currently, but you could always upgrade from old version to latest version without extra operations by hand if you use our `2.x` (current master). If breaking changes occur, we will bump version to 3.x (and upgrade guide will be provided).
+ Features in development are always in `Pull Request`, once done, the feature will be merged into master directly (sometimes it may cause some bugs indeed, we may try to avoid it).

I suggest that you use master and always update your version to our latest. If we have breaking changes again, we may change our strategy of our version management (multiple-release maybe).

This issue will be closed.Feel free to reopen if you have any further problem.

**Comment by lapo-luchini** [comment]
Upgrade from 1.x to latest can be done using dump/import and a tiny bit of `sed`ing. 😇
If you need to and fail, feel free to ask.
